src/bin/js/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Alt-Ergo JS</title>
+    <meta name="description" content="Alt-Ergo Js">
+    <meta name="author" content="Alt-Ergo dev team">
+    <link rel="stylesheet" href="style.css">
+    <!-- ENTRY POINT -->
+    <script type="text/javascript" src="alt-ergo-worker.js"></script>
```

Comment:
do you really need to include the worker here ?

Reply:
No, it was debug info to ensure the worker was accessible. It is not needed anymore, thanks to point it out